doc/troubleshooting_alerts.md

```diff
@@ -0,0 +1,101 @@
+# Troubleshooting Alerts
+
+You've been pinged by New Relic. Don't panic! Here are the next steps to
+take to work out what's going on, who to tell and how to fix it.
+
+## Jump into Slack
+
+You're likely not the only person who's seen the alert. Coordination will
+happen on Slack, so get on there first and see what people have ascertained
+already, to avoid duplication of work and stepping on each others' toes.
+
+## Acknowledge the alert
+
+The New Relic incident page (or the page on the mobile app) should have a
+big **Acknowledge** button. **Only press it once you're certain that the
+alert is being worked on** (by you or someone else).
+
+## Remember to close the NR incident afterwards
+
+This may seem a little early to mention it, but it's **really important**:
+if the incident isn't closed once it's fixed then new ones won't be created
+by later problems, and new alerts won't go out. This has bitten us before.
```

Comment:
Ouch. This strikes me as a real problem. Because, multiple incidents can happen in real life, and we need alerts for all of them. Also, humans make mistakes. We shouldn't base the reliability of our alert reporting on "Always remember to do XYZ!!"